guides/overview-apps.md

```diff
@@ -1,15 +1,3 @@
-# Wix App APIs
+# Apps made by Wix
```

Comment:
Our style is that for titles of articles, we capitalize the main words (not prepositions). 
For any other heading level, we use sentence case. 

So this line should be: "Apps Made by Wix" 

See the [style guide here](https://wix-private.github.io/tw-cafe/style-guide/grammar-formatting#headings).